### PR TITLE
Add admin pointer for document settings dropdown

### LINKED ISSUE #395: Add admin pointer to documents dropdown
It might not be completely clear that the document title in the document settings serves as a dropdown to switch between documents. As stated here: https://wordpress.org/support/topic/disable-packing-slips-3/#post-15849183

A solution to this would be to add an admin pointer that will display when the document settings page is loaded for the first time, and can then be dismissed.

### PR DESCRIPTION
**Pull request by Terminator-Barbapapa**
Closes #395

### CODE REVIEW

File: assets/js/admin-script.js [hunk 1]
```diff
@@ -51,6 +51,41 @@ jQuery( function( $ ) {
...
+					jQuery.post(
+						ajaxurl,
+						{
+							pointer: key,
+							action:  'dismiss-wp-pointer',
+						}
+					);
+				},
+			}
+		)
```
Comment (alexmigf) [inline assets/js/admin-script.js:81]: `;` missing

File: includes/class-wcpdf-assets.php [hunk 2]
```diff
@@ -148,6 +148,19 @@ public function backend_scripts_styles ( $hook ) {
 						'disable_free',
 						'use_latest_settings',
 					) ),
+					'pointers'                  => array(
+						'wcpdf_document_settings_sections' => array(
+							'target'        => '.wcpdf_document_settings_sections',
+							'content'       => sprintf( '<h3>%s</h3><p>%s</p>', __( 'Document settings', 'woocommerce-pdf-invoices-packing-slips'), __( 'Select a document in the dropdown menu above to edit its settings.', 'woocommerce-pdf-invoices-packing-slips') ),
```
Comment (alexmigf) [inline includes/class-wcpdf-assets.php:154]: WPCS

File: assets/js/admin-script.js [hunk 3]
```diff
@@ -51,6 +51,41 @@ jQuery( function( $ ) {
...
+						align: pointer.position.align
+					},
+	
+				pointerClass: pointer.pointer_class,
+	
+				pointerWidth: pointer.pointer_width,
+	
+				close: function() {
+					jQuery.post(
+						ajaxurl,
```
Comment (alexmigf) [inline assets/js/admin-script.js:73]: `wpo_wcpdf_admin.ajaxurl`

File: includes/class-wcpdf-assets.php (hunk 2 above)
Comment (Terminator-Barbapapa) [reply includes/class-wcpdf-assets.php:154]: @alexmigf Are you referring here to the `sprintf()` set on one line?

Comment (alexmigf) [reply includes/class-wcpdf-assets.php:154]: Yes, here `'woocommerce-pdf-invoices-packing-slips')` => `'woocommerce-pdf-invoices-packing-slips' )`, 2 times.

Comment (YordanSoares) [reply includes/class-wcpdf-assets.php:154]: @Terminator-Barbapapa it's the missing space after the text domains.

File: assets/js/admin-script.js (hunk 1 above)
Comment (Terminator-Barbapapa) [reply assets/js/admin-script.js]: Fixed here: 0f27d3b1610a13740cac524270dcc48acd2e8ddb

File: includes/class-wcpdf-assets.php (hunk 2 above)
Comment (Terminator-Barbapapa) [reply includes/class-wcpdf-assets.php]: Fixed here: 4cbd187db9404e3b01f33f3c61150f333e44b56a

File: assets/js/admin-script.js (hunk 3 above)
Comment (Terminator-Barbapapa) [reply assets/js/admin-script.js]: Fixed here: 7717c1e4509d18b778e82e689fb5adbf4df105b2

**Review by alexmigf — APPROVED** [review]

**Pull request merged**